Added Python Hex-to-String :)

EJBarba opened the pull request:
**Please describe your program and how to run it.**

An array of integers (in base 16) is converted into a string(to reveal its hidden message ;) ). 
In the working directory, just type python3 ejbarba_hex_to_string.py in your terminal to run :)

--------
**What Programming Language?**

Python

wzhouwzhou committed:
Add filename enforcement check

Commit:
Added Python Hex-to-String :)

wzhouwzhou commented on CONTRIBUTORS.md:
```diff
@@ -322,7 +323,7 @@ Place: São Paulo, SP, Brazil </br>
 Email: beatriz.uezu@gmail.com </br>
 
-﻿Name: [Hugo Nunes](https://github.com/hugonun)<br/>
+Name: [Hugo Nunes](https://github.com/hugonun)<br/>
```
To avoid merge conflicts, would you mind reversing these lines?

EJBarba replied:
@wzhouwzhou I have edited the file already, hopefully it works

EJBarba replied:
@wzhouwzhou i have merged your develop branch to my branch, maybe it would solve the merge conflicts, if not, I can just revert it to the previous commit

wzhouwzhou replied:
Seems like it's still trying to update files like _assemble/check.coffee

wzhouwzhou replied:
Do you think you can just make a new branch, copy only ejbarba_hex_to_string, and only your bio (without touching other people's work), and just make a clean PR?

EJBarba committed:
Update CONTRIBUTORS.md

Im not sure how to exactly fix the merge conflicts, but i hope this resolves it

Commit:
Merge remote-tracking branch 'upstream/develop' into feature/python-hex-to-string

wzhouwzhou reviewed (changes requested):
Please unmerge the develop branch, sorry I was not clear enough about that. We're currently using the develop branch as a launchpad for some organisational updates to roll out. CONTRIBUTORS.md doesn't seem like it was updated in the diff here. If all else fails, I would simply edit Contributors.md via the web just to respect what other people wrote.

wzhouwzhou commented:
If you want an example on how we resolved a similar situation, #351 is an excellent one.

Commit:
Revert "Update CONTRIBUTORS.md"

This reverts commit 9d59a9464803b88c51f4b09ec5f3f50435052c34.

EJBarba commented:
@wzhouwzhou  I have reverted unmerged the develop branch, I hope it resolves the problem

kannagikazuko reviewed (commented):
This diff is a mess, going to be closing the PR. @EJBarba, I suggest you re-open with a new branch or fork altogether.
